extensions/amp-story/0.1/amp-story.js

```diff
@@ -570,7 +574,9 @@ export class AmpStory extends AMP.BaseElement {
           });
         })
         .then(() => this.switchTo_(initialPageId))
-        .then(() => this.preloadPagesByDistance_());
+        .then(() => this.preloadPagesByDistance_())
+        // TODO(gmajoulet): preload only on mobile.
```

Comment:
nit: this means preload the share menu only on mobile, right?  can you clarify in the comment?  it's confusing since it comes right after `preloadPagesByDistance`, which (AFAIK) we want to do on mobile and desktop.